examples/array-map-symmetry/array_map_symmetry.bal

```diff
@@ -0,0 +1,43 @@
+import ballerina/io;
+
+public function main() {
+    // List constructors are used to construct list values.
+    // Arrays are lists with a uniform member type.
+    string[] list = ["foo", "bar"];
+
+    // Mapping constructors are used to construct map values.
+    // Maps are mapping with a uniform member type.
+    map<int> mapping = {x: 1, y: 2};
+
+    // Indexing is used with lists and mappings to access members.
+    // Index type of lists is `int`.
+    string listMember = list[0];
+    io:println(listMember);
+
+    // Index type of mappings is `string`.
+    int? mapIndex = mapping["x"];
+    io:println(mapIndex);
+
+    // A tuple type can be used to define per-index member types in a list.
+    [int, string, boolean] _ = [1, "John", true];
+
+    // A record type can be used to define per-index member types in a mapping.
+    record {
+        int id;
+        string name;
+        boolean checked;
+    } _ = {id: 1, name: "John", checked: true};
```

Comment:
Let's not ignore (use `_`) in BBEs. We can use a variable, and then use member and/or field access and show how the type is the exact type depending on the index or the key.